api4/license_test.go

```diff
@@ -15,22 +17,22 @@ func TestGetOldClientLicense(t *testing.T) {
 	license, resp := Client.GetOldClientLicense("")
 	CheckNoError(t, resp)
 
-	if len(license["IsLicensed"]) == 0 {
-		t.Fatal("license not returned correctly")
-	}
+	require.NotEqual(t, license["IsLicensed"], 0, "license not returned correctly")
```

Comment:
This should be
```suggestion
	require.NotEqual(t, license["IsLicensed"], "", "license not returned correctly")
```

Reply:
Ah yes, you are absolutely right; fixed.